app/services/queries/reviews_turnaround_per_user_query.rb

```diff
@@ -0,0 +1,28 @@
+module Queries
+  class ReviewsTurnaroundPerUserQuery < BaseService
+    attr_reader :time_interval
+
+    def initialize(time_interval:)
+      @time_interval = time_interval
+    end
+
+    def call
+      run
+    end
+
+    private
+
+    def query
+      Events::Review.joins(:review_request, pull_request: :project).includes(:review_request)
+                    .where(opened_at: @time_interval.starting_at..@time_interval.ending_at)
+    end
+
+    def run
+      query.collect do |review|
+        { user_id: review.owner.id,
```

Comment:
We want to track the review turnaround on each specific project, right? So this should be the id of the `UserProject` entity

Reply:
Yes! My bad, already fixed.